tests/runner.py

```diff
@@ -911,6 +926,34 @@ def get_bullet_library(runner_core, use_cmake):
       new_args += [arg]
   sys.argv = new_args
 
+  # Create a list of all known tests so that we can choose from them based on a wildcard search
+  all_tests = []
+  suites = ['default', 'other', 'browser', 'sanity', 'sockets', 'interactive']
```

Comment:
These names come from the global variables in the files. But that gave me an idea to simplify a bit more to avoid the special case, should look better now.